src/content/docs/dropins/all/localizing.mdx

```diff
@@ -7,10 +7,62 @@ sidebar:
 ---
 
 import Aside from '@components/Aside.astro';
+import Diagram from '@components/Diagram.astro';
+import CodeImport from '@components/CodeImport.astro';
+import CodeInclude from '@components/CodeInclude.astro';
+import dictionary from '@dropins/tools/types/elsie/src/i18n/en_US.json.d.ts?raw';
+import Vocabulary from '@components/Vocabulary.astro';
 
 In our global economy, localization is crucial. And it's one of the easiest changes to make. This page shows how to add a second language file and use it in your storefront.
 
-<Aside type="wip" title="Contribute!">
-  We need your help! If you have experience with drop-in component localization, please consider contributing
-  to this page.
-</Aside>
+## Big picture
+
+The following diagram shows the process for adding and overriding labels and text for drop-in components within the boilerplate template.
+
+<Diagram caption="How localization and labeling works in storefronts.">
+  ![How localization and labeling works in storefronts.](@images/DropinDictionaries.svg)
+</Diagram>
+
+## Vocabulary
+
+<Vocabulary>
+
+### Dictionaries
+
+JSON files that define the shape, keys, and default values for labels and text within a drop-in component. Each drop-in component has a default dictionary file: `en_US.json`. But drop-in components can contain multiple dictionary files as the site is localized for other languages, including `fr_CA.json` for French, `de_DE.json` for German, and so on.
+
+### Language code
+
+A two-letter code that identifies a language. For example, `en` for English, `fr` for French, and `de` for German.
+
+### Language files
+
+See dictionaries.
+
+### Locale
+
+A combination of a language code and a country code. For example, `en_US` for English in the United States, `fr_CA` for French in Canada, and `de_DE` for German in Germany.
+
+### Localization
+
+The process of adapting a product or service to a particular language.
+
+### Placeholders
+
+A Google Sheets file (`placeholders`) or a SharePoint Excel file (`placeholders.xlsx`) where you enter your site's text variables, which includes localization. The file has two columns: a _Key column_ that contains the variables in your site's code and a _Text column_ that contains the text values that will replace those variables at runtime. See [placeholders](https://www.aem.live/docs/placeholders) in the Edge Delivery Services documentation for more information.
```

Comment:
Should we talk about key format? @fnhipster recently did some work there in the boilerplate and I think he also may have found some inconsistencies between how we define placeholder keys and how EDS defines them.

Reply:
I updated the steps based on the latest added by @fnhipster to the `product-details.js` block in the `main` branch.